setup/terraform/terraform.go

```diff
@@ -39,6 +39,10 @@ func (t *Terraform) Get(dir string) error {
 	return t.run(dir, "get", "-update")
 }
 
+func (t *Terraform) Init(dir string) error {
+	return t.run(dir, "init", "-no-color")
```

Comment:
[nit] I don't think we need to explicitly set `-no-color` here, it makes it seem like things will break if someone were to remove it.